net/http/src/THttpLongPollEngine.cxx

```diff
@@ -27,6 +28,14 @@
 
 const char *THttpLongPollEngine::gLongPollNope = "<<nope>>";
 
+THttpLongPollEngine::QueueItem::~QueueItem()
+{
+   if (len && buf)
+      free((void *)buf);
```

Comment:
Please not :-(

Reply:
Could we agree to change this in following PR?
It has consequence on many other http classes with exactly such API.